app/biotracker/models/target.py

```diff
@@ -0,0 +1,45 @@
+class Target:
```

Comment:
Target is like a wrapper. Detection is something internal that we use to read/write to csv. Supposedly it has less overhead